bcda/main.go

```diff
@@ -40,6 +41,15 @@ type bulkResponseBody struct {
 	Errors              []fileItem `json:"error"`
 }
 
+func init() {
+	file, err := os.OpenFile("/var/log/bcda.log", os.O_CREATE|os.O_WRONLY, 0666)
```

Comment:
Same deal here -- please allow setting the log location via env var like `BCDA_ERROR_LOG`.